doc/nrf/introduction.rst

```diff
@@ -146,3 +146,11 @@ For more information about the west tool, see the :ref:`zephyr:west` user guide.
 
 See :ref:`getting_started` for information about how to install the |NCS| and about the first steps.
 See :ref:`dev-model` for more information about the |NCS| code base and how to manage it.
+
+.. _repository_revisions:
```

Comment:
Please remove this change, already a part of https://github.com/nrfconnect/sdk-nrf/pull/6848

Reply:
done